includes/class-wc-calypso-bridge-setup.php

```diff
@@ -680,6 +686,46 @@ public function prevent_redirects_on_activation( $location, $status ) {
 	private function write_to_log( $operation, $message ) {
 		error_log(  'WooExpress: Operation: (' . microtime( true ) . ') ' . $operation . ': ' . print_r( $message, 1 ) );
 	}
+
+	/**
+	 * Delete page by slug
+	 *
+	 * @param string $slug Slug.
+	 * @param string $operation Operation.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return void
+	 */
+	private function delete_page_by_slug( $slug, $operation ) {
+
+		$page = get_page_by_path( $slug, ARRAY_A );
+
+		if ( is_array( $page ) && isset( $page['ID'] ) ) {
+
+			$page_gmt_ts = get_post_time( 'U', true, $page['ID'] );
+			// draft pages don't have a post_date_gmt, so we need to calculate it.
+			if ( false === $page_gmt_ts ) {
+				$page_gmt_ts = get_gmt_from_date( $page['post_date'], 'U' );
+			}
+			$current_time_gmt_ts = current_time( 'U', true );
+			$diff_ts             = $current_time_gmt_ts - $page_gmt_ts;
+
+			if ( $diff_ts > 10 * MINUTE_IN_SECONDS ) {
```

Comment:
Do we want to ignore old Headstart pages as well? Note that we can look at the `_hs_old_id` `post_meta` field if we do want to delete Headstart pages in all cases.

Reply:
Same principle as in my previous comment; 

If headstart pages are old, we don't know if they have been modified for whatever reason. So if they are old, we keep them. 

What do you think?

Reply:
That makes sense -- once they're too old, we don't want to make any assumptions about those pages.